doc/source/groupby.rst

```diff
@@ -497,6 +497,28 @@ index are the group names and whose values are the sizes of each group.
 
    ``nth`` can act as a reducer *or* a filter, see :ref:`here <groupby.nth>`
 
+   Decimal columns are "nuisance" columns that .agg automatically excludes in groupby.
+
+   If you do wish to aggregate them you must do so explicitly:
+
+.. ipython:: python
+
+    from decimal import Decimal
+    dec = pd.DataFrame(
+            {'name': ['foo', 'bar', 'foo', 'bar'], 
+                'title': ['boo', 'far', 'boo', 'far'], 
+                'id': [123, 456, 123, 456], 
+                'int_column': [1, 2, 3, 4], 
+                'dec_column1': [Decimal('0.50'), Decimal('0.15'), Decimal('0.25'), Decimal('0.40')], 
+                'dec_column2': [Decimal('0.20'), Decimal('0.30'), Decimal('0.55'), Decimal('0.60')]
+            },
+        columns=['name','title','id','int_column','dec_column1','dec_column2']
+        )
+
+    dec.groupby(['name', 'title', 'id'], as_index=False).sum()
```

Comment:
add a comment before each of the groupbys (e.g. ``.sum()`` by default excludes nuiscance), and how to include